tests/test_project.py

```diff
@@ -72,16 +83,77 @@ def test_subpackage(baked_with_development_dependencies):
     if IS_WINDOWS_CI:
         # On Windows CI python and pip executable are in different paths
         bin_dir = ''
-    build_output = run([f'{bin_dir}python', 'setup.py', 'build'], project_dir)
-    assert build_output.returncode == 0
+    result = run([f'{bin_dir}python', 'setup.py', 'build'], project_dir)
+    assert result.returncode == 0
     assert (project_dir / 'build' / 'lib' / 'my_python_package' / 'mysub' / '__init__.py').exists()
     assert (project_dir / 'build' / 'lib' / 'my_python_package' / 'mysub' / 'mysub2' / '__init__.py').exists()
 
 
-def test_generate_api_docs(baked_with_development_dependencies):
-    project_dir, bin_dir = baked_with_development_dependencies
+def test_generate_api_docs(baked_with_development_dependencies, project_env_bin_dir):
+    project_dir = baked_with_development_dependencies
+    bin_dir = project_env_bin_dir
 
-    build_output = run([f'{bin_dir}sphinx-build', '-b', 'html', 'docs', 'docs/_build/html'], project_dir)
-    assert build_output.returncode == 0
-    assert 'build succeeded' in build_output.stdout
+    result = run([f'{bin_dir}sphinx-build', '-b', 'html', 'docs', 'docs/_build/html'], project_dir)
+    assert result.returncode == 0
+    assert 'build succeeded' in result.stdout
     assert (project_dir / 'docs' / '_build' / 'html' / 'index.html').exists()
+
+
+def test_coverage_api_docs(baked_with_development_dependencies, project_env_bin_dir):
+    project_dir = baked_with_development_dependencies
+    bin_dir = project_env_bin_dir
+
+    result = run([f'{bin_dir}sphinx-build', '-b', 'coverage', 'docs', 'docs/_build/coverage'], project_dir)
+    assert result.returncode == 0
+    assert 'build succeeded' in result.stdout
+    coverage_file = project_dir / 'docs' / '_build' / 'coverage' / 'python.txt'
+    lines = coverage_file.readlines()
+    assert len(lines) == 2
```

Comment:
Can you replace readlines with read and compare the content?

Reply:
done